content/views.py

```diff
@@ -20,9 +20,34 @@
 from content.permissions import IsArticleOwnerOrReadOnly, IsEditorOrReadOnly
 
 
+def transform_issue_name(term):
+    matches = re.match(r"(?:v?(\d+))?(?:i([0-9A-Z]+))?", term, flags=re.I)
+    volume = matches[1]
+    issue = matches[2]
+    if volume is not None and issue is not None:
+        return Q(volume_num=volume) & Q(issue_num=issue)
+    elif volume is not None:
+        return Q(volume_num=volume)
+    elif issue is not None:
+        return Q(issue_num=issue)
+    else:
+        return ~Q(pk__in=[])
+
+
 class IssueViewSet(ModelViewSet):
     queryset = Issue.objects.all()
     serializer_class = IssueSerializer
+    filter_backends = [SearchableFilterBackend]
+    search_fields = []
+    search_transformers = {
+        "__term": transform_issue_name
+    }
+
+    class __PseudoArticleViewSet:
```

Comment:
Not a particularly huge fan of this approach, would it be possible to add an entry point to the filter logic that allows us to manually pass in the needed parameters instead of getting them off a "view" object?

This probably won't be the last time this pattern will come up.

Reply:
*I had something written up, but then I was dumb and did some things and now I lost it, so this will be a bit terse.*

I was envisioning that this endpoint would eventually be replaced with a search filter. That being said, I think I can override the constructor and possibly bypass this hack. I've made a commit (85a52590) to test this, can you see if it works?

Reply:
update: managed to test it on user, and it works perfectly.